.nais/send-varsler-job-dev.yaml

```diff
@@ -7,12 +7,17 @@ metadata:
     team: team-esyfo
 spec:
   image: {{ image }}
-  schedule: "*/15 * * * *"
+  schedule: "*/5 * * * *"
+  azure:
+    application:
+      enabled: true
```

Comment:
Hvorfor må man legge til azure i jobben? Er ikke den eneste den gjør å kalle endepunktet for å starte sending av varsler?